### PR TITLE
Check server version

### PR DESCRIPTION
**Pull request by chaws**
Fixes https://github.com/Linaro/squad-client/issues/84

### CODE REVIEW

**Comment by amrohassaan** [comment]
@chaws i think there is a conflict that needs resolving. Other than this, LGTM :+1:

File: squad_client/commands/version.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import logging
+
+from squad_client.core.api import SquadApi
+from squad_client.core.command import SquadClientCommand
+from squad_client.version import __version__
+
+
+logger = logging.getLogger()
```
Comment (jscook2345) [inline squad_client/commands/version.py:8]: Please use `__name__` for the logger name.

ex: `logger = logging.getLogger(__name__)`

**Comment by jscook2345** [comment]
@chaws 

Besides the conflict it LGTM as well! Thanks!

**Comment by chaws** [comment]
Rebased it against master, merging now

**Pull request merged**